systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ConnectorsTest.java

```diff
@@ -0,0 +1,332 @@
+
+    private static Logger log = CustomLogger.getLogger();
+
+    private final String remoteBrokerNamespace = "systemtests-external-broker";
+    private final String remoteBrokerUsername = "test-user";
+    private final String remoteBrokerPassword = "test-password";
+    private Endpoint remoteBrokerEndpoint;
+    private Endpoint remoteBrokerEndpointNoSSL;
+
+    @BeforeEach
+    public void deployBroker() throws Exception {
+        SystemtestsKubernetesApps.deployAMQBroker(remoteBrokerNamespace, remoteBrokerUsername, remoteBrokerPassword);
+        remoteBrokerEndpoint = SystemtestsKubernetesApps.getAMQBrokerSSLEndpoint(remoteBrokerNamespace);
+        remoteBrokerEndpointNoSSL = SystemtestsKubernetesApps.getAMQBrokerEndpoint(remoteBrokerNamespace);
+        log.info("Endpoints to remote broker:");
+        log.info("Route with SSL: {}", remoteBrokerEndpoint);
+        log.info("Service without SSL: {}", remoteBrokerEndpointNoSSL);
+    }
+
+    @AfterEach
+    public void undeployBroker(ExtensionContext context) throws Exception {
+        if (context.getExecutionException().isPresent()) { //test failed
+            logCollector.collectLogsOfPodsInNamespace(remoteBrokerNamespace);
+            logCollector.collectEvents(remoteBrokerNamespace);
+        }
+        SystemtestsKubernetesApps.deleteAMQBroker(remoteBrokerNamespace);
+    }
+
+    @Test
+    public void testBrokerDeployment() throws Exception {
+        AmqpClient client = createClientToRemoteBroker();
+
+        Address testQueue = new AddressBuilder()
+                .withNewMetadata()
+                .withName("testtt")
+                .endMetadata()
+                .withNewSpec()
+                .withAddress("queue1")
+                .endSpec()
+                .build();
+        QueueTest.runQueueTest(client, testQueue);
+        client.close();
+    }
+
+    @Test
+    public void testBrokerDeploymentNonSSL() throws Exception {
+        AmqpClient client = createClientToRemoteBrokerNoSSL();
+
+        Address testQueue = new AddressBuilder()
+                .withNewMetadata()
+                .withName("testtt")
+                .endMetadata()
+                .withNewSpec()
+                .withAddress("queue1")
+                .endSpec()
+                .build();
+        QueueTest.runQueueTest(client, testQueue);
+        client.close();
+    }
+
+    @Test
+    public void testSendThroughConnector1() throws Exception {
+        doTestSendThroughConnector("*");
+    }
+
+    @Test
+    public void testSendThroughConnector2() throws Exception {
+        doTestSendThroughConnector("queue/*");
+    }
+
+    @Test
+    public void testReceiveThroughConnector1() throws Exception {
+        doTestReceiveThroughConnector("*");
+    }
+
+    @Test
+    public void testReceiveThroughConnector2() throws Exception {
+        doTestReceiveThroughConnector("queue/*");
```

Comment:
This test is failing because the pattern is remote/queue/* whereas the test tries to send to the addresses remote/queue1 and remote/queue2 .